operators/endpointmetrics/controllers/observabilityendpoint/ocp_monitoring_config.go

```diff
@@ -603,7 +607,7 @@ func createOrUpdateCMOConfig(
 		return false, err
 	}
 
-	if equality.Semantic.DeepEqual(*existingCfg, *updatedCMOCfg) {
+	if reflect.DeepEqual(*existingCfg, *updatedCMOCfg) {
```

Comment:
Why this change? Earlier the the PR it was changed the other way (from reflect->equality).